### ISSUE TITLE
Could not find generator 'talking_stick:views'.

### ISSUE THREAD

**Issue by suramai** [mojolingo/talking_stick#20]
Hi,

Could you please help in solving this issue?

```
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ rails generate talking_stick:views
Running via Spring preloader in process 188485
Could not find generator 'talking_stick:views'. Maybe you meant 'assets', 'migration' or 'test_unit:job'
Run `rails generate --help` for more options.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ bundle exec spring binstub --all
* bin/rake: spring already present
* bin/rails: spring already present
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ rails generate talking_stick:views
Running via Spring preloader in process 188527
Could not find generator 'talking_stick:views'. Maybe you meant 'assets', 'migration' or 'test_unit:job'
Run `rails generate --help` for more options.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ 
```

Thanks

**Comment by bklang** [comment]
Not sure why this is not working for you. Would you include a gist of your Gemfile.lock? Also, please let me know what version of Ruby you are running.

**Comment by suramai** [comment]
Hello @bklang 

This is my Gemfile.lock

```
GEM
  remote: https://rubygems.org/
  specs:
    actionmailer (4.2.5)
      actionpack (= 4.2.5)
      actionview (= 4.2.5)
      activejob (= 4.2.5)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 1.0, >= 1.0.5)
    actionpack (4.2.5)
      actionview (= 4.2.5)
      activesupport (= 4.2.5)
      rack (~> 1.6)
      rack-test (~> 0.6.2)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (4.2.5)
      activesupport (= 4.2.5)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    activejob (4.2.5)
      activesupport (= 4.2.5)
      globalid (>= 0.3.0)
    activemodel (4.2.5)
      activesupport (= 4.2.5)
      builder (~> 3.1)
    activerecord (4.2.5)
      activemodel (= 4.2.5)
      activesupport (= 4.2.5)
      arel (~> 6.0)
    activesupport (4.2.5)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    arel (6.0.3)
    binding_of_caller (0.7.2)
      debug_inspector (>= 0.0.1)
    builder (3.2.2)
    byebug (8.2.1)
    coffee-rails (4.1.1)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.1.x)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.10.0)
    concurrent-ruby (1.0.0)
    debug_inspector (0.0.2)
    erubis (2.7.0)
    execjs (2.6.0)
    globalid (0.3.6)
      activesupport (>= 4.1.0)
    i18n (0.7.0)
    jbuilder (2.4.0)
      activesupport (>= 3.0.0, < 5.1)
      multi_json (~> 1.2)
    jquery-rails (4.0.5)
      rails-dom-testing (~> 1.0)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    json (1.8.3)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.3)
      mime-types (>= 1.16, < 3)
    mime-types (2.99)
    mini_portile2 (2.0.0)
    minitest (5.8.3)
    multi_json (1.11.2)
    nokogiri (1.6.7.1)
      mini_portile2 (~> 2.0.0.rc2)
    rack (1.6.4)
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.2.5)
      actionmailer (= 4.2.5)
      actionpack (= 4.2.5)
      actionview (= 4.2.5)
      activejob (= 4.2.5)
      activemodel (= 4.2.5)
      activerecord (= 4.2.5)
      activesupport (= 4.2.5)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.2.5)
      sprockets-rails
    rails-deprecated_sanitizer (1.0.3)
      activesupport (>= 4.2.0.alpha)
    rails-dom-testing (1.0.7)
      activesupport (>= 4.2.0.beta, < 5.0)
      nokogiri (~> 1.6.0)
      rails-deprecated_sanitizer (>= 1.0.1)
    rails-html-sanitizer (1.0.2)
      loofah (~> 2.0)
    railties (4.2.5)
      actionpack (= 4.2.5)
      activesupport (= 4.2.5)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (10.4.2)
    rdoc (4.2.1)
      json (~> 1.4)
    sass (3.4.20)
    sass-rails (5.0.4)
      railties (>= 4.0.0, < 5.0)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    sdoc (0.4.1)
      json (~> 1.7, >= 1.7.7)
      rdoc (~> 4.0)
    spring (1.6.1)
    sprockets (3.5.2)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.0.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sqlite3 (1.3.11)
    talking_stick (0.0.1)
      jquery-rails (~> 4.0)
      rails (~> 4.0)
    thor (0.19.1)
    thread_safe (0.3.5)
    tilt (2.0.1)
    turbolinks (2.5.3)
      coffee-rails
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uglifier (2.7.2)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    web-console (2.2.1)
      activemodel (>= 4.0)
      binding_of_caller (>= 0.7.2)
      railties (>= 4.0)
      sprockets-rails (>= 2.0, < 4.0)

PLATFORMS
  ruby

DEPENDENCIES
  byebug
  coffee-rails (~> 4.1.0)
  jbuilder (~> 2.0)
  jquery-rails
  rails (= 4.2.5)
  sass-rails (~> 5.0)
  sdoc (~> 0.4.0)
  spring
  sqlite3
  talking_stick
  turbolinks
  uglifier (>= 1.3.0)
  web-console (~> 2.0)
```

This is the Gemfile

```
source 'https://rubygems.org'


# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.2.5'
# Use sqlite3 as the database for Active Record
gem 'sqlite3'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.1.0'
# See https://github.com/rails/execjs#readme for more supported runtimes
# gem 'therubyracer', platforms: :ruby

# Use jquery as the JavaScript library
gem 'jquery-rails'
# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
gem 'turbolinks'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.0'
# bundle exec rake doc:rails generates the API under doc/api.
gem 'sdoc', '~> 0.4.0', group: :doc

# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Unicorn as the app server
# gem 'unicorn'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> in views
  gem 'web-console', '~> 2.0'

  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end



gem 'talking_stick'
```

**Comment by benlangfeld** [comment]
Try `spring stop`.

**Comment by suramai** [comment]
Hello @benlangfeld 

Thanks. I tried the following.

```
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ spring stop
Spring stopped.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ rails generate talking_stick:views
Running via Spring preloader in process 189004
Could not find generator 'talking_stick:views'. Maybe you meant 'assets', 'migration' or 'test_unit:job'
Run `rails generate --help` for more options.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ spring stop
Spring stopped.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ rails generate talking_stick:views
Running via Spring preloader in process 189030
Could not find generator 'talking_stick:views'. Maybe you meant 'assets', 'migration' or 'test_unit:job'
Run `rails generate --help` for more options.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ spring stop
Spring stopped.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ spring stop
Spring is not running
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ rails generate talking_stick:views
Running via Spring preloader in process 189060
Could not find generator 'talking_stick:views'. Maybe you meant 'assets', 'migration' or 'test_unit:job'
Run `rails generate --help` for more options.
suramai@rails-tutorial:~/workspace/testingwebrtc (master) $ 
```

Not working.

**Comment by bklang** [comment]
Ok, this one is our fault. You're using the release version of the gem, which is badly out of date.

Quick fix: Change your Gemfile to read `gem 'talking_stick', github: 'mojolingo/talking_stick'` and re-run `bundle install`. This will use the Gem directly from Github.
Our fix: we will release a new version of Talking Stick that has the features you need.

**Comment by benlangfeld** [comment]
@bklang Anything required beyond the standard release process?

**Comment by bklang** [comment]
Nope, just doing some last minute checks before making the release

**Comment by suramai** [comment]
Thanks @bklang and @benlangfeld. Working now :)

**Comment by bklang** [comment]
Fixed with the recent release of version 1.0: (#21)
